source/_patterns/02-organisms/components/hero-banner~blog-article.json

```diff
@@ -0,0 +1,38 @@
+{
+  "image": {
+    "sources": [
+      {
+        "srcset": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+        "type": "image/jpeg"
+      }
+    ],
+    "fallback": {
+      "defaultPath": "https://unsplash.it/1114/336",
+      "altText": ""
+    }
+  },
+  "subjects": {
+    "list": [
+      {
+        "name": "Biophysics and Structural Biology",
+        "url": "#"
+      }
+    ]
+  },
+  "title": "Media coverage: Slime can see",
+  "titleLength": "large",
+  "url": "#",
+  "summary": "In their research paper – Cyanobacteria use micro-optics to sense light direction – Schuergers et al. reveal how bacterial cells act as the equivalent of a microscopic eyeball or the world/'s oldest and smallest camera eye, allowing them to /'see/'.",
+  "meta": {
+    "url": "#",
+    "text": "Research article",
```

Comment:
This should be `Inside eLife`.

Reply:
The meta text is changes to Inside eLife